README.md

```diff
@@ -166,543 +119,10 @@ There are two parameters:
-	if l, err = d.DecodeSliceLen(); err != nil {
-		return err
-	}
-	if l != 2 {
-		return fmt.Errorf("array len doesn't match: %d", l)
-	}
-	if m.Name, err = d.DecodeString(); err != nil {
-		return err
-	}
-	if m.Val, err = d.DecodeUint(); err != nil {
-		return err
-	}
-	return nil
-}
-
-func (c *Tuple) EncodeMsgpack(e *msgpack.Encoder) error {
-	if err := e.EncodeSliceLen(3); err != nil {
-		return err
-	}
-	if err := e.EncodeUint(c.Cid); err != nil {
-		return err
-	}
-	if err := e.EncodeString(c.Orig); err != nil {
-		return err
-	}
-	if err := e.EncodeSliceLen(len(c.Members)); err != nil {
-		return err
-	}
-	for _, m := range c.Members {
-		e.Encode(m)
-	}
-	return nil
-}
-
-func (c *Tuple) DecodeMsgpack(d *msgpack.Decoder) error {
-	var err error
-	var l int
-	if l, err = d.DecodeSliceLen(); err != nil {
-		return err
-	}
-	if l != 3 {
-		return fmt.Errorf("array len doesn't match: %d", l)
-	}
-	if c.Cid, err = d.DecodeUint(); err != nil {
-		return err
-	}
-	if c.Orig, err = d.DecodeString(); err != nil {
-		return err
-	}
-	if l, err = d.DecodeSliceLen(); err != nil {
-		return err
-	}
-	c.Members = make([]Member, l)
-	for i := 0; i < l; i++ {
-		d.Decode(&c.Members[i])
-	}
-	return nil
-}
-
-func main() {
-	// establish connection ...
-
-	tuple := Tuple{777, "orig", []Member{{"lol", "", 1}, {"wut", "", 3}}}
-	_, err = conn.Replace(spaceNo, tuple)  // NOTE: insert structure itself
-	if err != nil {
-		t.Errorf("Failed to insert: %s", err.Error())
-		return
-	}
-
-	var tuples []Tuple
-	err = conn.SelectTyped(spaceNo, indexNo, 0, 1, IterEq, []interface{}{777}, &tuples)
-	if err != nil {
-		t.Errorf("Failed to SelectTyped: %s", err.Error())
-		return
-	}
-
-	// same result in a "magic" way
-	var tuples2 []Tuple2
-	err = conn.SelectTyped(spaceNo, indexNo, 0, 1, IterEq, []interface{}{777}, &tuples2)
-	if err != nil {
-		t.Errorf("Failed to SelectTyped: %s", err.Error())
-		return
-	}
-
-	// call function 'func_name' returning a table of custom tuples
-	var tuples3 []Tuple
-	err = client.CallTyped("func_name", []interface{}{1, 2, 3}, &tuples3)
-	if err != nil {
-		t.Errorf("Failed to CallTyped: %s", err.Error())
-		return
-	}
-}
-
-/*
-// Old way to register types
-func init() {
-	msgpack.Register(reflect.TypeOf(Tuple{}), encodeTuple, decodeTuple)
-	msgpack.Register(reflect.TypeOf(Member{}), encodeMember, decodeMember)
-}
-
-func encodeMember(e *msgpack.Encoder, v reflect.Value) error {
-	m := v.Interface().(Member)
-	// same code as in EncodeMsgpack
-	return nil
-}
-
-func decodeMember(d *msgpack.Decoder, v reflect.Value) error {
-	m := v.Addr().Interface().(*Member)
-	// same code as in DecodeMsgpack
-	return nil
-}
-
-func encodeTuple(e *msgpack.Encoder, v reflect.Value) error {
-	c := v.Interface().(Tuple)
-	// same code as in EncodeMsgpack
-	return nil
-}
-
-func decodeTuple(d *msgpack.Decoder, v reflect.Value) error {
-	c := v.Addr().Interface().(*Tuple)
-	// same code as in DecodeMsgpack
-	return nil
-}
-*/
-
-```
-
-## Options
-
```

Comment:
* `Timeout` - timeout for any particular request. If `Timeout` is zero request,
  any request can be blocked infinitely,
* `Reconnect` - timeout between reconnect attempts. If `Reconnect` is zero, no
  reconnect attempts will be made,
* `MaxReconnects` - maximum number of reconnect failures; after that we give it
  up to on. If `MaxReconnects` is zero, the client will try to reconnect endlessly,
* `User` - username for logging in to Tarantool,
* `Pass` - user password for logging in to Tarantool.

Reply:
Updated.